src/components/Branding/BrandingStyles.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingStyles = styled.div`
+  position: relative;
+  width: 100%;
+  text-align: center;
+  padding: 16px;
+  span:nth-child(1) {
+    color: ${color.dark};
```

Comment:
You don't need to specify color.dark here, just hardcode body {color: [hard code color here]} in defaults.css